messages/proposed_event.go

```diff
@@ -1,14 +1,108 @@
 package messages
 
 import (
+	"encoding/json"
+
 	uuid "github.com/gofrs/uuid"
 )
 
 // ProposedEvent ...
 type ProposedEvent struct {
```

Comment:
I didn't notice "ProposedEvent" term in other language's clients. I also didn't found any information about that in EventStrore docs. Maybe it should be just `Event`?

Reply:
That is true that other client uses `EventData`. `ProposedEvent` event is actually the wording we wanted initially. Believe it or not but the Go client was among the first ESDB gRPC clients! That name has stayed there ever since. `Event` is too generic.

One of the perks of using `ProposedEvent` (an event that we send to the server) is it's consistent with the other types of event, namely:

`RecordedEvent` and `ResolvedEvent`.

Reply:
ok, fair enough :ok_hand: